examples/v2/sqladmin/README.md

```diff
@@ -0,0 +1,36 @@
+# Cloud SQL Example
+
+## Overview
+
+This is a [Google Cloud Deployment
+Manager](https://cloud.google.com/deployment-manager/overview) template that
+deploys a Cloud SQL instance and database.
+
+## Deploy the template
+
+Use `sqladmin.yaml` to deploy this example template. When ready, deploy
+with the following command:
+
+    gcloud deployment-manager deployments create my-database --config sqladmin.yaml
+
+## Deleting the root user
+
+If you want to delete the root user, you can add the following resource.
```

Comment:
This is already in the jinja, I think it should be added to the python template as well. Documented in the overview and then this section can be removed.

Reply:
Ah whoops. That was actually an accident, but it probably is a better practice to have folks delete the root user by default.